cli/dts/lib.deno.unstable.d.ts

```diff
@@ -510,6 +512,10 @@ declare namespace Deno {
    *                      `deno run`. If sources are provided, it should match
    *                      one of the names of the sources.
    * @param options  A set of options to be used with the emit.
+   * 
+   * @returns The result of the emit. If diagnostics are found, they can be used
+   * with `Deno.formatDiagnostics` to construct a user friendly string, which
+   * has the same format/result `tsc` will provide.
```

Comment:
nick pick, but replace `will` with `would`?

> The same result tsc will provide
> The same result tsc would provide

Reply:
It's technically slightly different as Deno has its own diagnostic formatting implementation. Maybe just drop the "which has the same format/result `tsc` will provide." since the rest of the sentence seems sufficient?

Reply:
I think this is what should be pointed out:
```suggestion
   * has the same format as CLI type errors.
```

Reply:
If that, then "as CLI diagnostics". Could be more than just type errors.

Reply:
> i'll hold off on actioning this then until the proper wording has been agreed on :P

It's not that kind of conundrum, just use the last suggestion.

Reply:
https://github.com/denoland/deno/pull/10925/commits/4c9613523c10c6138305cc8cbef63eb5b83cc77e